pkg/networking/cilium/templater.go

```diff
@@ -237,6 +237,11 @@ func templateValues(spec *cluster.Spec, versionsBundle *cluster.VersionsBundle)
 		val["egressMasqueradeInterfaces"] = spec.Cluster.Spec.ClusterNetwork.CNIConfig.Cilium.EgressMasqueradeInterfaces
 	}
 
+	if spec.Cluster.Spec.ClusterNetwork.CNIConfig.Cilium.RoutingMode == anywherev1.CiliumRoutingModeDirect {
+		val["tunnel"] = "disabled"
+		val["routingMode"] = "native"
```

Comment:
EKS-A uses Cilium v1.12 and will be transitioning to v1.13. Neither of these have the `routingMode` option option so I don't think this will be successful.

Reply:
You're right the flag only got added in 1.14, i had the wrong branch open. I have added the NativeRouting prefix to the config and set the auto direct mode accordingly